website/src/pages/showcase/_components/ShowcaseCard/index.tsx

```diff
@@ -65,16 +64,20 @@ function ShowcaseCard({user}: {user: User}) {
   return (
     <li key={user.title} className="card shadow--md">
       <div className={clsx('card__image', styles.showcaseCardImage)}>
-        <Image img={image} alt={user.title} />
+        <Image img={image} alt={user.title ?? 'Project preview'} />
       </div>
       <div className="card__body">
         <div className={clsx(styles.showcaseCardHeader)}>
           <Heading as="h4" className={styles.showcaseCardTitle}>
-            <Link href={user.website} className={styles.showcaseCardLink}>
+            <Link
+              href={user.website}
+              className={styles.showcaseCardLink}
+              target="_blank"
+              rel="noopener noreferrer">
               {user.title}
             </Link>
           </Heading>
-          {user.tags.includes('favorite') && (
+          {Array.isArray(user.tags) && user.tags.includes('favorite') && (
```

Comment:
There's no reason to do `Array.isArray(user.tags)` if we know it's an array already